### PR TITLE
Unbind event channel before closing it

### PR DESCRIPTION
**Pull request by benjamreis**
Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

**Comment by benjamreis** [comment]
Event channel is bound [here](https://github.com/xcp-ng/uefistored/blob/master/src/uefistored.c#L804-L812).

**Review by andSmv — APPROVED** [review]

**Comment by benjamreis** [comment]
Will test before merge.

**Review by stormi — APPROVED** [review]

File: src/uefistored.c [hunk 1]
```diff
@@ -571,7 +580,7 @@ int main(int argc, char **argv)
 {
     xc_dominfo_t domain_info;
     shared_iopage_t *shared_iopage;
-    size_t vcpu_count = 1;
+    vcpu_count = 1;
```
Comment (Wescoeur) [inline src/uefistored.c:583]: You can move the initialization directly where `vcpu_count` is declared in the global namespace.

**Comment by benjamreis** [comment]
XCp-ng tests successful on Alpine, Debian10 & Win10! :)

**Comment by Wescoeur** [comment]
Well, I think it's a better idea to split the commit:
  - One with the new std flag.
  - Another with the clean of the channel.
  
Otherwise you can merge. :+1:

**Review by Wescoeur — APPROVED** [review]

**Pull request merged**